test/node/features/restify.feature

```diff
@@ -1,3 +1,4 @@
+@skip_node_18
```

Comment:
the current restify fixture doesn't work on node 18. It has been verified manually that these tests pass with an updated fixture with the latest restify but it has been left to a separate task to adjust the e2e tests to add additional fixtures to cover restify on all supported node versions